DoePaAdminDataModel/DTO/RemainingBudgetOnOrdersDTO.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DoePaAdminDataModel.DTO
+{
+    public class RemainingBudgetOnOrdersDTO
```

Comment:
I think you should use this class to satisfy the "The datagrid's data can be exported to a csv- or xlsx-file (xlsx is optional)" requirement. Therefore you would need to add at least the order's name, the order item's name.

Reply:
Okay forget about that. I have to think about that a little longer.